lighthouse-core/report/report-generator.js

```diff
@@ -261,14 +262,16 @@ class ReportGenerator {
     });
 
     const template = Handlebars.compile(this.getReportTemplate());
+    reportContext = reportContext || 'extension'; // could be: devtools, extension
```

Comment:
ah, that's fine (and I just noticed you were just moving it up from where it already was). Maybe just add a note about the default to the jsdoc?